values.yaml

```diff
@@ -534,6 +534,11 @@ server:
     path: "/v1/sys/health?standbyok=true"
```

Comment:
It would be great to add a comment for `path` too, now that we also have `exec`.

Reply:
Will update it. Thank you!